public/data/MadhuSaini22.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "Madhu Saini",
+  "type": "personal",
+  "bio": "Hello, I'm Madhu Saini. I am an Enthusiastic Open-Source contributor, Web Developer, Currently diving straight in Open Source and believing in collaboration instead of competition.",
```

Comment:
```suggestion
  "bio": "Hello, I'm Madhu Saini. I am an Enthusiastic Open-Source contributor, Web Developer, currently diving straight in Open Source and believing in collaboration instead of competition.",
```